lib/events-reader-checkpoint-writer.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+const EventEmitter = require('events');
+const _ = require('lodash');
+let writerLoopStopped = true;
+let lastDoc;
+let lastCheckpointId;
+let harvesterApp;
+let writeInterval = 1;
+const log = require('agco-logger')({});
+const shouldLog =  (process.env.LOG_EVENTS_READER=="true") ||false;
```

Comment:
What do you think about moving const declaration to the top of the file? You could also add a space after the `||` operator here